apis/Google.Cloud.Speech.V1/smoketests.json

```diff
@@ -5,10 +5,11 @@
       "config": {
       	"language_code": "en-US",
       	"sample_rate_hertz": 44100,
-      	"encoding": "FLAC"
+        "encoding": "FLAC",
+        "audio_channel_count": 2
```

Comment:
Fix indentation to be consistent, here and for the uri in the other file? I'm not massively bothered, admittedly...